tabled_derive/src/parse/field_attr.rs

```diff
@@ -15,6 +15,7 @@ pub fn parse_field_attributes(
 }
 
 pub struct FieldAttr {
+    #[allow(dead_code)]
```

Comment:
why choose to allow `dead code`?

Reply:
I'd like to hear from @zhiburt on this.
I see this field was added in https://github.com/zhiburt/tabled/commit/ba048dbc22922ae0584a21d83a6ce10f43dd099e. I know it was a long time ago, but do you remember why it exists but never read or it's fine if I remove it?

As for those `#[allow(dead_code)]` that I left in the test code, I think the reason why they are there is more understandable - they are intended for fields that we never read in tests, but should be able to write to them

Reply:
> As for those #[allow(dead_code)] that I left in the test code, I think the reason why they are there is more understandable - they are intended for fields that we never read in tests, but should be able to write to them

correct;
though these test could be cleaner I guess

> if I remove it?

Will be all right.